.dockerignore

```diff
@@ -4,7 +4,10 @@
 dashboards/
 hooks/
 out/
-tests/
+tests/db
+tests/e2e
+tests/queries
+tests/unit
```

Comment:
Are we aware that we're leaving the `tests/data` directory outside of `.dockerignore`?

Reply:
Yes, that was the intention. The DB tests need this data because Dockerfile.db uses it.

Reply:
Probably not the best solution, but the only one that would make the db tests pass locally. I hope this is ok for now.